test/FunctionalTests/Tests/DataEdmLib/FunctionalTests/VocabularyValidationTests.cs

```diff
@@ -425,6 +423,151 @@ public void ValidateOutOfLineAnnotationWithValidTargetsInReferencedModel()
             Assert.IsTrue(validationResult, "Expected no validation errors from annotation with targets in an external model.");
         }
 
+        [TestMethod]
+        public void ValidateUnresolvedInlineAnnotationTargets()

```

Comment:
>public void ValidateUnresolvedInlineAnnotationTargets() [](start = 8, length = 55)

The test cases seem not related to nav prop bindings?

Reply:
Yes; I ran across several other issues in trying to fix navpropbindings, and in order to get a build out that included all of the fixes I put them in the same PR.

---
In reply to: [174972862](https://github.com/OData/odata.net/pull/1109#discussion_r174972862) [](ancestors = 174972862)